### ISSUE TITLE
Support color blind single stat thresholds

### ISSUE THREAD

**Issue by pcarranza** [grafana/grafana#13014]
This is not an issue of something that is broken, but rather something I just realized.

When building a singlestat panel the provided colors for thresholds are red, yellow and red. This makes sense to the vast majority of people, and it should stay this way.

But, I remember once building in a system that had to provide some form of feedback to people to let them know when things weren't working fine. We used colors, like everybody else does, but then one person in the organization approached to me to explain that he was color blind and that what I was doing was basically useless for him, because he saw no difference between them. I was humbled by this simply because I didn't even consider this to be a thing (the blessings of being in the majority side, I guess).

I took a look at the issue tracker and community and failed to find anything about color blind support.

So, I was thinking that grafana could do something really simple to support this: use stripes, solid, or dots, or whatever it makes sense, but have a way of differentiating thresholds by something else than the color itself.

Something as simple as this, with a tick for "enable color blind" would include all this people who we (at least I) don't usually consider.

**Comment by mpursley** [comment]
I'm thinking that if I was color-blind, I would probably just choose very high contrast colors for the thresholds (e.g. dark, medium, bright).  Then I would just pay attention to the very bright metrics on the panels, and not so much the darker ones.

Though, those colors would be save within the dashboard.  So, everyone would have to have the same "color blind" view for this dashboard.  Maybe there could be a way to update the colors used for all the single stat thresholds on the page at once?  Then I could select the "color blind" view when viewing the dashboard, but others could leave it in the normal "green, orange, red" mode.  Something like that might be even more useful?  

![image](https://user-images.githubusercontent.com/3343322/55924416-aa43e180-5bbe-11e9-9d7b-effa7d26acf5.png)
